components/brave_rewards/extension/brave_rewards/components/panel.tsx

```diff
@@ -204,7 +204,9 @@ export class Panel extends React.Component<Props, State> {
     const publisher: RewardsExtension.Publisher | undefined = this.getPublisher()
     const converted = utils.convertBalance(balance.toString(), rates)
     const notification = this.getNotification()
-
+    if (publisher && publisher.provider && publisher.provider.toLowerCase() === 'youtube') {
```

Comment:
let's use the same logic that we use for contribute table where we have support for regular sites, youtube and twitch.

https://github.com/brave/brave-core/blob/master/components/brave_rewards/ui/components/contributeBox.tsx#L49

Reply:
note that we currently have only favicon for twitch VOD's and not for live videos.

Reply:
Fixed